recipe/meta.yaml

```diff
@@ -127,7 +119,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage('ray-core', exact=True) }}
-        - aiohttp 3.7
+        - aiohttp >=3.7
```

Comment:
- smart_open may be added (see https://github.com/ray-project/ray/blob/ray-1.9.2/python/setup.py#L212)